crates/oxc_transformer/src/options.rs

```diff
@@ -1,8 +1,27 @@
+use oxc_syntax::assumptions::CompilerAssumptions;
+
+use crate::{es2020::NullishCoalescingOperatorOptions, react_jsx::ReactJsxOptions};
+
 #[derive(Debug, Default, Clone, Copy)]
 pub struct TransformOptions {
```

Comment:
It feels like we can add a a `From<TransformTarget>` and "from preset-env" here later.